.github/workflows/generator.yml

```diff
@@ -12,11 +12,8 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Checkout problem-specifications
-        uses: actions/checkout@v3
-        with:
-          repository: exercism/problem-specifications
-          path: .problem-specifications
+      - run: bin/fetch-configlet
+      - run: bin/configlet sync || true
```

Comment:
A clever way to check out problem-specs. Since you really just want that side-effect so you might want to add a comment to indicate that pulling the problem-specs to the cache directory is used by subsequent `run` commands.